### DISCUSSION TITLE
Improvements on `helmfile list`

### DISCUSSION THREAD

**Discussion by lucasfcnunes** [helmfile/helmfile#1550] [Ideas]
I use `helmfile list --environment ${ENV} --output json | yq -P` (for `${ENV}` in `yq '.environments[] | key'`) and I miss some fields like `environment` and `kubeContext` on the output.

What about...
1. `helmfile list | yq -I=0 -o json '.[0] | keys'` outputting `["environment","kubeContext","name","namespace","enabled","installed","labels","chart","version"]` (new keys `environment` and `kubeContext`)?
1. `helmfile list --output yaml`?
1. `helmfile list` (or `helmfile list -e '.*'`) loop through all environments by default?
1. add `helmfile (path)` field too?

**Comment by yxxhero** [comment]
@lucasfcnunes PR is welcome.
